Bio/PDB/parse_pdb_header.py

```diff
@@ -15,10 +16,50 @@
 
 
 import re
+from collections import defaultdict
 
 from Bio import File
 
 
+def _get_biomoltrans(inl):
+    # REMARK 350
+    # REMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN
+    # REMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE
+    # REMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS
+    # REMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND
+    # REMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.
+    # REMARK 350
+    # REMARK 350 BIOMOLECULE: 1
+    # REMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC
+    # REMARK 350 APPLY THE FOLLOWING TO CHAINS: A
+    # REMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000
+    # REMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000
+    # REMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000
+    biomolecule = defaultdict(list)
+    for line in inl:
+        if re.search(r"\AREMARK 350", line):
```

Comment:
Would it simpler to just do `if line.startswith(...)` instead of calling `re.search`?

Reply:
Yes